services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_utils.py

```diff
@@ -95,6 +99,118 @@ def all_containers_running(containers_inspect: List[DockerContainerInspect]) ->
     )
 
 
+async def service_remove_containers(
+    dynamic_sidecar_client: DynamicSidecarClient,
+    scheduler_data: SchedulerData,
+    progress_callback: Optional[ProgressCallback] = None,
+) -> None:
+    try:
+        await dynamic_sidecar_client.stop_service(
+            scheduler_data.endpoint, progress_callback=progress_callback
+        )
+    except (BaseClientHTTPError, TaskClientResultError) as e:
+        logger.warning(
+            (
+                "Could not remove service containers for "
+                "%s\n%s. Will continue to save the data from the service!"
+            ),
+            scheduler_data.service_name,
+            f"{e}",
+        )
+
+
+async def service_save_state(
+    dynamic_sidecar_client: DynamicSidecarClient,
+    dynamic_sidecar_endpoint: AnyHttpUrl,
+    progress_callback: Optional[ProgressCallback] = None,
+) -> None:
+    await dynamic_sidecar_client.save_service_state(
+        dynamic_sidecar_endpoint, progress_callback=progress_callback
+    )
+
+
+async def service_push_outputs(
+    dynamic_sidecar_client: DynamicSidecarClient,
+    dynamic_sidecar_endpoint: AnyHttpUrl,
+    progress_callback: Optional[ProgressCallback] = None,
+) -> None:
+    await dynamic_sidecar_client.push_service_output_ports(
+        dynamic_sidecar_endpoint, progress_callback=progress_callback
+    )
+
+
+async def service_remove_docker_resources(
```

Comment:
I was confused by what you name "docker resources". So this is actually the docker stack correct?
Could you maybe use the same naming as docker? for me resources sounds like a anything...